app/_locales/en/messages.json

```
@@ -749,7 +749,7 @@
     "message": "Contract deployment"
   },
   "contractDescription": {
-    "message": "To protect yourself against scammers, take a moment to verify contract details."
+    "message": "To protect yourself against scammers, take a moment to verify third party details."
```

I believe @coreyjanssen has suggested to use a hyphen here. @coreyjanssen I'm a little confused as to when to use a hypen. Could you double check all the content in this file but expanding it and reading the green highlighted sections.
```suggestion
    "message": "To protect yourself against scammers, take a moment to verify third-party details."
```

---

Will do, reviewing the green text now! 

It's a weird rule but basically when` third party` is used as a modifier to a word like` details,` you use a hyphen. Otherwise, it's a noun with no hyphen. So `third-party details` vs `third party`, if that makes sense?